index.js

```diff
@@ -149,27 +154,29 @@ async function loadSigningKey() {
 
 /**
  * Get the public key in JWK format
+ * @param {*} env
  * @returns
  */
-async function loadPublicKey() {
+async function loadPublicKey(env) {
   // if the JWK values are already in KV then just return that
-  const key = await KV.get(KV_SIGNING_KEY, 'json')
+  const key = await env.KV.get(KV_SIGNING_KEY, 'json')
   if (key) {
     return { kid: key.kid, ...key.public }
   }
 
   // otherwise generate keys and store the Keyset in KV
-  const { kid, publicKey } = await generateKeys()
+  const { kid, publicKey } = await generateKeys(env)
   return { kid, ...publicKey }
 }
 
 /**
  * Turn a payload into a JWT
  * @param {*} payload
+ * @param {*} env
  * @returns
  */
-async function signJWT(payload) {
-  const { kid, privateKey } = await loadSigningKey()
+async function signJWT(payload, env) {
```

Comment:
Its a nit, but can you make env the first argument for all of these functions. That way its consistent on being passed first through the file.